scripts/techsoup/us_usda/Summer_Meal_Sites_2022_v0.tmcf

```diff
@@ -0,0 +1,22 @@
+Node: E:Data->E1
+typeOf: UsdaSummerMealSite
+day: E:Data->E2
```

Comment:
I wonder if we want to link from UsdaSummerMealSiteDay to this as - `mealSite` or something

`day` seems a bit too general.